api/envoy/config/filter/http/cache/v2/cache.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+
+package envoy.config.filter.http.cache.v2;
+
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.config.filter.http.cache.v2";
+option java_outer_classname = "CacheProto";
+option java_multiple_files = true;
+
+// [#protodoc-title: HTTP Cache Filter]
+// Pluggable HTTP Cache :ref:`overview <config_http_filters_cache>`.
+// [#extension: envoy.filters.http.cache]
+message Cache {
```

Comment:
Nit: probably best to call this `CacheConfig` to avoid potential conflation.

Reply:
done